src/medCore/gui/database/medDatabaseView.cpp

```diff
@@ -559,13 +559,23 @@ void medDatabaseView::onEditRequested(void)
         QList<QVariant> attributes = item->attributes();
         QList<QVariant> values = item->values();
         QList<QString> labels;
-        
+
+        // The last attribute is ThumbnailPath. Users are not allowed to change it.
+        attributes.removeLast();
+        values.removeLast();
```

Comment:
It is last because in ```medDatabaseModel``` it is the last attribute added, and because the sorting algorithm preserves that order. That is two internal dependancies that this code is relying on to work. Change any of those and this will fail.